.yamato/Test Il2cpp-deps.yml

```diff
@@ -0,0 +1,115 @@
+# Creates an IL2CPP branch test-mono-il2cpp-deps-[unity-version] and updates stevedore / il2cpp deps file with
+# the latest commit hash from the corresponding Mono branch. Does not push the branch, but runs the IL2CPP
+# bee-update-verification tests on it 'locally'.
+# Intended to be used for testing Mono changes when used within IL2CPP (i.e. to test what 'Update Il2cpp-deps'
+# will do when run 'for real').
+name: Test Mono Il2cpp-deps [Manual] [Existing-Branch]
```

Comment:
I'd prefer a different name for this job. I'm thinking of something more descriptive, maybe "Run IL2CPP tests against this branch of Mono".

Reply:
Done